SumoLogic.Logging.NLog/BufferedSumoLogicTarget.cs

```diff
@@ -82,7 +80,7 @@ public BufferedSumoLogicTarget(ILog log, HttpMessageHandler httpMessageHandler)
             this.MaxQueueSizeBytes = 1000000;
             this.LogLog = log ?? new DummyLog();
             this.HttpMessageHandler = httpMessageHandler;
-            this.AppendException = true;
+            this.Layout = "${longdate}|${level:uppercase=true}|${logger}${exception:format=tostring}{newline}";

```

Comment:
Is there any particular reason to use this layout here?

Reply:
I have just taken the default NLog Layout and and added `${exception}{newline}` to the end. Because you have exception and newline by default (Just like the default log4net pattern).

Reply:
This need to be updated to the document as well as other default values